src/guide/frontend/android.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Android Client SDK
+---
+
+# Android Client SDK
+
+The Passwordless.dev Android client SDK gives users the ability to leverage their device’s built-in fingerprint sensor and/or FIDO security keys for secure passwordless access to websites and native applications that support the FIDO2 protocols
+
+## Requirements
+
+- Android 9.0 (API level 28) or higher
+- Java 8 or higher
+- [Having completed the 'Get started' guide](../get-started.md).
+
+## Installation
+
+::: tabs
+
+@tab Apache Maven
+
+```xml
+<dependency>
+  <groupId>com.bitwarden</groupId>
+  <artifactId>passwordless-android</artifactId>
+  <version>1.0.1</version>
+</dependency>
+```
+
+@tab Gradle Kotlin DSL
+
+```kotlin
+implementation("com.bitwarden:passwordless-android:1.0.1")
+```
+
+@tab Gradle Groovy DSL
+
+```groovy
+implementation 'com.bitwarden:passwordless-android:1.0.1'
+```
+
+:::
+
+## Permissions
+
+In your `AndroidManifest.xml`, you will need to add the following permissions:
+
+```xml
+<uses-permission android:name="android.permission.INTERNET" />
+```
+
+## Configuration (Android application)
+
+```kotlin
+data class PasswordlessOptions(
+   // Your public API key
+   val apiKey: String,
+
+   // Identifier for your server, for example 'example.com' if your backend is hosted at https://example.com.
+   val rpId: String,
+
+   // This is where your Facet ID goes
+   val origin: String,
+
+   // Where your backend is hosted
+   val backendUrl:String,
+
+   // Passwordless.dev server, change for self-hosting
+   val apiUrl: String = "https://v4.passwordless.dev"
+)
+```
+
+### .well-known/assetlinks.json
+
+In your application's `AndroidManifest.xml`, you will then need to add the tag below under `manifest::application`:
```

Comment:
Little confusing to read, perhaps "add the follow tag under manifest::application"